### PR TITLE
es6-modules | Keivan Mojmali

### PR DESCRIPTION
**Pull request by keivanmojmali**
image

### CODE REVIEW

File: es6-modules/src/lib/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import noop from './noop';
+import toArray from './to-array';
+import createElement from './create-element';
+
+export default {
```
Comment (scbowler) [inline es6-modules/src/lib/index.js:5]: Here you want to get rid of the `default` keyword. You want to create named exports.

File: es6-modules/src/components/todo-app.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import createElement from '../lib/create-element';
+import noop from "../lib/noop";
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js:2]: After removing `default` from 'lib/index.js` you take advantage of that file and import both of these on 1 line:

```JAVASCRIPT
import { createElement, noop } from '../lib';
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Really close, 2 fixes needed

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**